src/rai/rai/node.py

```diff
@@ -312,7 +366,7 @@ def get_raw_message_from_topic(self, topic: str, timeout_sec: int = 1) -> Any:
                     f"No message received in {timeout_sec} seconds from topic {topic}"
                 )
                 self.get_logger().error(error)
-                return error
+                return Exception(error)
```

Comment:
This is a no-no. Technically, you can, but you should never return an instance of `Exception` class in standard applications (maybe only just in mockups or some error-handling frameworks). 

If you want to distinguish success/false and don't raise Exceptions, then I see two approaches:
1. Return `None`, and check `if self.get_raw_message_from_topic():` for success/false.
2. Return a tuple `(message, success) = self.get_raw_message_from_topic()` and check `success` for true/false.